libs/vm-modules/tests/unit/state_tests.cpp

```diff
@@ -579,11 +579,11 @@ TEST_F(StateTests, test_serialisation_of_fixed_point32)
   ASSERT_TRUE(static_cast<bool>(retval));
   ASSERT_EQ(int32_t{3}, retval->Count());
 
-  EXPECT_EQ(fetch::fixed_point::fp32_t{1.0},
+  EXPECT_EQ(fetch::fixed_point::fp32_t("1.0"),
```

Comment:
please add tests to check scientific notation works as well

Reply:
ok

Reply:
scientific notation was removed after discussion.